andela/travel_tool_front
# #160141144 Update SideDrawer on mobile screen

#### What does this PR do?
Replaces static data on sidedrawer to logged-in user information on mobile screen

#### Description of Task to be completed?
* Connect SideDrawer component to redux-store
* Retrieve logged-in user information
* Replace static data with information retrieved

#### How should this be manually tested?
* Clone the repo - git clone https://github.com/andela/travel_tool_front.git
* Change into the project directory
* Install project dependencies: yarn install
* Start app using make start
* Log in and change to mobile view
* Click the hamburger icon on the top-left, you'll see the logged-in 
user information as it is on larger screen

#### Any background context you want to provide?
N/A

#### What are the relevant pivotal tracker stories?
[#160141144](https://www.pivotaltracker.com/story/show/160141144)

#### Screenshots (if appropriate)
<img width="233" alt="screen shot 2018-08-29 at 11 20 43 pm" src="https://user-images.githubusercontent.com/23608060/44818835-439b2600-abe2-11e8-9aac-90fc630c138a.png">


#### Questions:
